Boy or Girl/Boy or Girl/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Boy_or_Girl
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Reader reader = new Reader();
+            var userName = reader.Next();
+            var check = checkGirl(userName);
+            if (check == true)
```

Comment:
Check is a boolean value itself so we donot need to compare it with another boolean value.

Reply:
```suggestion
            var check = checkGirl(userName);
            if (check)
```

Reply:
Or we can shorten it as following suggestion:
```suggestion
            if (checkGirl(userName))
```